setup.md

```diff
@@ -1,7 +1,157 @@
+it will host an ESGF node where CMIP data is locally stored on disk and accessible directly by the tool;
+in the same manner, observational (OBS) data will be found centrally stored as well. Here are a few options
+for compute clusters with ESGF nodes:
+
+- CEDA-Jasmin (UK):
+- DKRZ (Germany):
+- ETHZ (Switzerland):
+
+
+### CEDA-Jasmin
+
+If you do not already have an account on JASMIN, then request an account as
+soon as possible. The instructions on how to create a Jasmin account are here: https://help.jasmin.ac.uk/article/4435-get-a-jasmin-account-portal
+
+During the account creation, you will need an SSH key, which can be generated following these instructions: https://help.jasmin.ac.uk/article/185-generate-ssh-key-pair 
+
+More instructions on how to get started:
+https://help.jasmin.ac.uk/article/189-get-started-with-jasmin 
+
+Also note that if you are working from home, JASMIN may not be directly
+accessible from your home. You may need to use ssh to connect to a machine
+in your institute and then on to JASMIN. Please test your connection before
+the tutorial. 
+
+#### Jasmin-login
+
+Note that you have only created an account for the web-interface. To log into the jasmin machine and do work, you&#39;ll need to create a login account too: https://help.jasmin.ac.uk/article/161-get-login-account 
+
+#### Access to data on JASMIN
+
+Please request access to the working groups:
+● https://accounts.jasmin.ac.uk/services/group_workspaces/esmeval 
+●https://services.ceda.ac.uk/cedasite/resreg/application?attributeid=cmip5_research 
+
+Once you have access to the data archive on CEDA, make sure to link your
+CEDA and JASMIN accounts. This can be done by checking the link to CEDA box on your JASMIN profile page:
+https://accounts.jasmin.ac.uk/account/profile/ 
+
+The linking may take a few hours to take effect and is necessary for you to
+access the BADC archives via JASMIN. Some CMIP5 data sets such as MIROC
+are not accessible by default and special permission has to be requested to
+access them via the CEDA catalogue page https://catalogue.ceda.ac.uk/
+
+#### Test your Setup
+Log into jasmin-login:
+ssh -X JASMIN-USERNAME@jasmin-login1.ceda.ac.uk 
+
+Then log into the sci1 machine:
+ssh -X jasmin-sci1
+
+Can you see the following locations:
+- ls /group_workspaces/jasmin4/esmeval/obsdata/Tier2
+- ls /badc/cmip5/data/cmip5/output1/MOHC/HadGEM2-ES
+- ls /badc/cmip6/data/CMIP6/CMIP/*/*/historical/r1i1p1f?/Omon/[ts]os/gn/latest/*.nc
+
+Note that the JASMIN is only open to certain locations (mostly universities, and research centres). You may need a VPN if you wish to connect from your home network.
+
+### DLR 
+FIXME
+
+### Other computing systems
 FIXME
 
+### Your own machine
+
+If you are planning on running ESMValTool on your own machine, please make sure that you are able to download CMIP data and that you have several GB of space available to install conda & ESMVAlTool, but also enough to make a copy of some data. 
```

Comment:
For someone without access to an ESGF node it would be nice if he/she could run a plot recipy without having to know which data he/she needs to download before. As ESMValTool can use [synda](http://prodiguer.github.io/synda/index.html) to download CMIP data. 

It would be nice to install synda in this section. Something like instructions at https://docs.esmvaltool.org/en/latest/input.html#models

Reply:
I've added thr synda stuff to the required packages, but it probably needs a lot more detail. In fact, the whole "your own machine" could use a bit of work.

Reply:
I created https://github.com/ESMValGroup/tutorial/issues/54 to improve the section later.